composer.json

```
@@ -6,27 +6,25 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.9",
+        "drupal/core": "^8.9 || ^9.1",
```

As we cannot be sure that our sites can be updated from 8.9 to 9.1 directly, without passing by 9.0, we have to make sure that a component will not add such a constraint, unless it is actually needed.

Please use here:

```suggestion
        "drupal/core": "^8.9 || ^9",
```